services/management/checks.go

```diff
@@ -73,3 +80,59 @@ func (s *ChecksAPIService) GetSecurityCheckResults() (*managementpb.GetSecurityC
 
 	return &managementpb.GetSecurityCheckResultsResponse{Results: checkResults}, nil
 }
+
+// ListSecurityChecks returns all available STT checks.
+func (s *ChecksAPIService) ListSecurityChecks() (*managementpb.ListSecurityChecksResponse, error) {
+	disChecks, err := s.checksService.GetDisabledChecks()
+	if err != nil {
+		s.l.Errorf("Failed to get disabled security checks list: %+v", err)
+		return nil, status.Error(codes.Internal, "Failed to get disabled checks list.")
+	}
+
+	m := make(map[string]struct{}, len(disChecks))
+	for _, c := range disChecks {
+		m[c] = struct{}{}
+	}
+
+	checks := s.checksService.GetAllChecks()
+	res := make([]*managementpb.SecurityCheck, 0, len(checks))
+	for _, c := range checks {
+		_, disabled := m[c.Name]
+		res = append(res, &managementpb.SecurityCheck{Name: c.Name, Disabled: disabled})
+	}
+
+	return &managementpb.ListSecurityChecksResponse{Checks: res}, nil
+}
+
+// ChangeSecurityChecks allows to change STT checks state.
+func (s *ChecksAPIService) ChangeSecurityChecks(req *managementpb.ChangeSecurityChecksRequest) (*managementpb.ChangeSecurityChecksResponse, error) {
+	var enableChecks, disableChecks []string
+	for _, check := range req.Params {
+		if check.Enable && check.Disable {
+			return nil, status.Errorf(codes.InvalidArgument, "Check %s has enable and disable parameters set to the true.", check.Name)
+		}
+
+		if check.Enable {
+			enableChecks = append(enableChecks, check.Name)
+		}
+
+		if check.Disable {
+			disableChecks = append(disableChecks, check.Name)
+		}
+	}
+
+	err := s.checksService.EnableChecks(enableChecks)
+	if err != nil {
+		s.l.Errorf("Failed to enable disabled security checks: %+v", err)
+		return nil, status.Error(codes.Internal, "Failed to enable disabled security checks.")
+	}
+
+	err = s.checksService.DisableChecks(disableChecks)
+	if err != nil {
+		s.l.Errorf("Failed to disable security checks: %+v", err)
```

Comment:
We _can_ just return raw `err` there, it will be logged in the interceptor, and generic internal error will be returned to the caller

But up to you

Reply:
Yep, good remark 👍

Done